### PR TITLE
Content loader header carousel

### PR DESCRIPTION
**Pull request by ahmedsamir96m**

### CODE REVIEW

**Comment by ahmedsamir96m** [comment]
There was no example suitable than this one.
if you have any ideas, kindly share them

**Comment by Tramsi** [comment]
> There was no example suitable than this one.
> if you have any ideas, kindly share them

I guess featured banner section is a good example ...we need to show a content holder for the text not the background image and this will also make it easier for the navbar section to be turned into content loader as well

**Comment by AbdoMgdy** [comment]
> There was no example suitable than this one.
> if you have any ideas, kindly share them

you are supposed to make your own using https://danilowoz.com/create-content-loader/

File: src/components/HeaderCarousel/index.js [hunk 1]
```diff
@@ -121,6 +125,7 @@ class HeaderCarousel extends Component {
               </div>
             )}
           </CarouselProvider>
+          <Loader width="1024" height="800" textAlign="center" />
```
Comment (AbdoMgdy) [inline src/components/HeaderCarousel/index.js:128]: i think this is a mistake? pls remove it

**Pull request merged**